docs/guides/how_to_guides/creating_and_editing_expectations/how_to_contribute_a_new_expectation_to_the_shared_library.rst

```diff
@@ -0,0 +1,148 @@
+.. _how_to_guides__creating_and_editing_expectations__how_to_template:
```

Comment:
I'm not reviewing this doc closely in this PR--I assume that we'll pick up any issues in UAT and iteration around the hackathons.